config/config_loader.go

```diff
@@ -91,6 +93,17 @@ func Load() {
 		}
 	}
 
+	var eventDispatcherType string
+	if consumerConfig != nil {
+		eventDispatcherType = consumerConfig.eventDispatcherType
+	}
+	// notice consumerConfig.eventDispatcherType will be replaced
+	if providerConfig != nil {
+		eventDispatcherType = providerConfig.eventDispatcherType
+	}
+	// init EventDispatcher
+	observer.SetAndInitGlobalDispatcher(eventDispatcherType)
```

Comment:
I think the better way to initialize the global dispatcher is making "direct" as the field `eventDispatcherType`'s default value. And then we can avoid checking the name inside the `SetAndInitGlobalDispatcher`.